Dacaspex/Exhibit
# #385 temporary table crash fix - POI metrics

fixes #385 
#### What have I done
Created a temporary fix for the table such that it does not crash when no POI metric information can be found. A proper fix will be implemented in #382  

#### Testing
1. Import some data
2. Open the table
3. Select a sheet
    - Observe that the table does not crash

#### Addresses SRD section(s)
/
